hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenRejectHandler.java

```diff
@@ -104,10 +149,10 @@ public void pureChecks(@NonNull final TransactionBody txn) throws PreCheckExcept
             validateAccountID(op.owner(), null);
         }
 
-        var uniqueTokenReferences = new HashSet<TokenReference>();
+        final var uniqueTokenReferences = new HashSet<TokenReference>();
```

Comment:
On line 147, Should we fail ? I don't think we fail for an empty `CryptoTransfer`

Reply:
Hmm I think we fail when the transfer list for CryptoTransfer is empty see [here](https://github.com/hashgraph/hedera-services/blob/5a04dc6f20538f4a711cba3d7d85ebab1cac9e2e/hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenCreateSpecs.java#L1170).